app/src/openxr/cpp/OpenXRInput.cpp

```diff
@@ -146,6 +146,14 @@ OpenXRInputMapping* OpenXRInput::GetActiveInputMapping() const
   return nullptr;
 }
 
+bool OpenXRInput::HasPhysicalControllersAvailable() const {
+    if (!AreControllersReady())
+        return false;
+    if (mInputSources[0]->IsUsingHandInteracionProfile())
+        return mInputSources[0]->NumberOfMappings() > 1;
+    return true;
```

Comment:
Instead exposing that low level info I think it's better if we have a method that just return a boolean and return that value directly here. The OpenXR input are generic and should not deal with input profiles and the like. The details should be implemented in the input source.

Reply:
Done.